packages/react-native-attestation/example/package.json

```diff
@@ -18,10 +19,13 @@
     "@babel/runtime": "^7.20.0",
     "@react-native/eslint-config": "^0.72.2",
     "@react-native/metro-config": "^0.72.11",
-    "metro-react-native-babel-preset": "0.76.8",
-    "babel-plugin-module-resolver": "^5.0.0"
+    "babel-plugin-module-resolver": "^5.0.0",
```

Comment:
Wondering out loud why we use `bable`. Though `tsc` can do everything it can do.

Reply:
This is a bob thing, it build typescript definition using `tsc` but the code transpiling is actually done using `babel`